open-machine-learning-jupyter-book/ml-advanced/ensemble-learning/stacking.ipynb

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
<p>Why is it a blank file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/757/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Im still working on the content.

Reply:
<p>Then there is no need to upload this file or add toc. </p>

Reply:
deleted